cmake/FindHiopHipLibraries.cmake

```diff
@@ -0,0 +1,54 @@
+#[[
+
+Exports target `hiop_hip` which finds all hip libraries needed by hiop.
+
+#]]
+
+add_library(hiop_hip INTERFACE)
+
+find_library(HIPBLAS_LIBRARY
+  NAMES
+  hipblas
+  PATHS
+  ENV LD_LIBRARY_PATH ENV DYLD_LIBRARY_PATH
+  PATH_SUFFIXES
+  lib64 lib)
+
+if(HIPBLAS_LIBRARY)
+  get_filename_component(HIPBLAS_LIBRARY_DIR ${HIPBLAS_LIBRARY} DIRECTORY)
+endif()
+
+find_path(HIPBLAS_INCLUDE_DIR
+  NAMES
+  hipblas.h
+  PATHS
+  ${HIPBLAS_LIBRARY_DIR}/..
+  PATH_SUFFIXES
+  include)
+
+if(HIPBLAS_LIBRARY)
+  add_library(Hipblas INTERFACE)
+  target_link_libraries(Hipblas INTERFACE ${HIPBLAS_LIBRARY})
+  target_include_directories(Hipblas INTERFACE ${HIPBLAS_INCLUDE_DIR})
+  message(STATUS "Found Hipblas include: ${HIPBLAS_INCLUDE_DIR}")
+  message(STATUS "Found Hipblas library: ${HIPBLAS_LIBRARY}")
+else()
+  message(STATUS "Hipblas was not found.")
+endif()
+
+# for now we rely on MAGMA for GPUs computations
+include(FindHiopMagma)
+target_link_libraries(Magma    INTERFACE Hipblas)
```

Comment:
This line is causing an issue with exportingn as Hipblas is linked against, but not a part of an export set. Adding:

```CMake
install(TARGETS Hipblas EXPORT hiop-targets)
```

should fix :)